pkg/webhook/resources/upgrade/validator.go

```diff
@@ -238,6 +246,29 @@ func (v *upgradeValidator) checkManagedCharts() error {
 	return nil
 }
 
+// Since volume snapshot/backup will hold one additional LH VA tickets, it can block VM live migration.
```

Comment:
how about extend the code in the function too: https://github.com/FrankYang0529/harvester/blob/9d1180fa081a5d367d56c8ad0a437862868688d0/pkg/webhook/resources/upgrade/validator.go#L409

Reply:
Sounds good to me. Updated it. Thanks for the suggestion.